### ISSUE TITLE
Budget tree colors

### ISSUE THREAD

**Issue by jagthedrummer** [codeforokc/openbudgetokc#89]
Can we get more consistent colors within agencies?

**Comment by dashcraft** [comment]
@jagthedrummer Something like a color wheel, differing opacities, or anything in particular you have in mind?

The current earth tone colors are a little hard on the eyes at times.

**Comment by jagthedrummer** [comment]
Yep, agreed. I'm open to anything really. This was just to capture some ideas from a meeting. Suggestions are more than welcome.

**Comment by dashcraft** [comment]
Gotcha! I may take a little time with my ui guy when i get the chance and get his opinion, i'll check out dribble and behance and see if any color combos look good.

I think i saw you at the techlahoma town hall meeting today btw, nice to meet you!

**Comment by smurphy8** [comment]
@dashcraft any progress here?

**Comment by dashcraft** [comment]
Thank you for getting back to me, hit me up on slack for quicker responses.

I did investigate several different color schemas, these are the ones
i personally like!

https://yeun.github.io/open-color/
http://color.koya.io/
